### PR TITLE
Each subclass was triggering code injection for it's parent class

### PR DESCRIPTION
**Pull request by tlaport4**
**IMPORTANT**: By submitting a patch, you agree that your work will be licensed under the [license](https://github.com/umple/umple/blob/master/LICENSE.md) used by the project.

If your pull request does not pass CI, it will **not** be merged (unless _heavily_ justified). 
## Description

Please provide a description of what this PR is meant to fix, and how it works (if it's not clear from the code changes). 
## Tests

Please list all major test changes. All issues should have a test created to address the issue in the future (as relevant). 

If closing issues, please write `Closes #ISSUE_NUMBER[, #ISSUE_NUMBER2, ...]`, to automatically close the equivalent issue.

**Note:** This template is a guide, remove sections as applicable.

### CODE REVIEW

File: cruise.umple/src/Generator_CodeJava.ump [hunk 1]
```diff
@@ -1934,7 +1851,112 @@ class JavaGenerator
...
+  	prepareCodeInjectionForAssociations(aClass);
+  }
   
+  private void prepareCodeInjectionForAttributes(UmpleClass aClass)
+  {
+  	for(Attribute av : aClass.getAttributes())
+    {
+      prepareCodeInjectionForAttribute(aClass, av);
+    }
+  }
```
Comment (aforward) [inline cruise.umple/src/Generator_CodeJava.ump:1866]: Can you ensure all indentation are spaces (2 for an indent).  I am seeing some tabs in this PR

**Comment by tlaport4** [comment]
Sure, I'll update when I get a chance.

I'm used to using tabs, sorry.

File: cruise.umple/src/Generator_CodeJava.ump (hunk 1 above)
Comment (tlaport4) [reply cruise.umple/src/Generator_CodeJava.ump:1866]: Sure, I'll update when I get a chance.

I'm used to using tabs, sorry.

**Comment by vahdat-ab** [comment]
I forced the server to rebuild it again on Windows. We'll see how it goes.

**Comment by TimLethbridge** [comment]
I'm not bothered by the appveyor failing on the PR. It passed on the branch. That should be good enough to merge, but I will leave it to Vahdat

**Comment by vahdat-ab** [comment]
@TimLethbridge I was trying to understand the issue. We somehow need to know why this happens. It shows the error related to getting http://downloads.sourceforge.net/project/simpletest/simpletest/simpletest_1.1/simpletest_1.1.0.tar.gz
I consider this as a timeout issue.

**Pull request merged**